src/test/javascript/spec/component/participation-submission/participation-submission.component.spec.ts

```diff
@@ -271,28 +271,31 @@ describe('ParticipationSubmissionComponent', () => {
             jest.spyOn(participationService, 'find').mockReturnValue(of(new HttpResponse({ body: participation1 })));
         });
 
-        afterEach(() => {
-            expect(comp.submissions?.length).toBe(1);
-            expect(comp.submissions![0].results?.length).toBe(1);
-            expect(comp.submissions![0].results![0]).toEqual(result1);
-        });
-
         it('should delete result of fileUploadSubmission', fakeAsync(() => {
             jest.spyOn(exerciseService, 'find').mockReturnValue(of(new HttpResponse({ body: fileUploadExercise })));
             deleteResult(submissionWithTwoResults, result2);
             flush();
+            expect(comp.submissions?.length).toBe(1);
+            expect(comp.submissions![0].results?.length).toBe(1);
+            expect(comp.submissions![0].results![0]).toEqual(result1);
```

Comment:
```suggestion
            expect(comp.submissions).toHaveLength(1);
            expect(comp.submissions![0].results).toHaveLength(1);
            expect(comp.submissions![0].results![0]).toEqual(result1);
```

The same is valid for the occurrences of this that you pasted below; also, can we extract these three lines into a function instead of duplicating the code?